Production/crab/config/QCD.txt

```diff
@@ -0,0 +1,31 @@
+EventAwareLumiBased
+globalTag=80X_mcRun2_asymptotic_2016_TrancheIV_v6 sampleType=Summer16MC ReRunJEC=True applyTriggerMatch=True saveGenTopInfo=False saveGenBosonInfo=True saveGenJetInfo=True
+
+QCD_Pt_1000to1400           20000	/QCD_Pt_1000to1400_TuneCUETP8M1_13TeV_pythia8/RunIISummer16MiniAODv2-PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6-v1/MINIAODSIM
```

Comment:
Why 20000? I expect it will still be a lot of small jobs. Let's put 150000, as the data. We can decrease it later.